app/views/shared/_users_gardens_index.html.erb

```diff
@@ -7,15 +7,18 @@
         <section class='garden' id='garden-<%= garden.id %>'>
           <h3><%= link_to garden.name, garden_path(garden.id) %></h3>
           <div class='garden-image'>
-            <%= image_tag 'default-garden.png' %>
+            <a href="<%=garden_path(garden.id)%>">
+              <img alt="Image-<%= garden.id %>" src="/assets/default-garden-5bbbceb5c8def07b7a99e836154844af647a4598ec1250edaf878c68467caad9.png">
+
+            </a>
```

Comment:
Nice. What if we created an array of image links and chose randomly to give the stock photos some variation? Just a thought and probably an extension